Octokit.Tests/Models/SearchIssuesRequestTests.cs

```diff
@@ -25,5 +27,156 @@ public void SortNotSpecifiedByDefault()
             Assert.True(string.IsNullOrWhiteSpace(request.Sort));
             Assert.False(request.Parameters.ContainsKey("sort"));
         }
+
+        [Fact]
+        public void HandlesStringAttributesCorrectly()
+        {
+            var stringProperties = new List<Tuple<string, Action<SearchIssuesRequest, string>>>()
+            {
+                new Tuple<string, Action<SearchIssuesRequest, string>>("author:", (x,value) => x.Author = value),
+                new Tuple<string, Action<SearchIssuesRequest, string>>("assignee:", (x,value) => x.Assignee = value),
+                new Tuple<string, Action<SearchIssuesRequest, string>>("mentions:", (x,value) => x.Mentions = value),
+                new Tuple<string, Action<SearchIssuesRequest, string>>("commenter:", (x,value) => x.Commenter = value),
+                new Tuple<string, Action<SearchIssuesRequest, string>>("involves:", (x,value) => x.Involves = value),
+                new Tuple<string, Action<SearchIssuesRequest, string>>("team:", (x,value) => x.Team = value),
+                new Tuple<string, Action<SearchIssuesRequest, string>>("head:", (x,value) => x.Head = value),
+                new Tuple<string, Action<SearchIssuesRequest, string>>("base:", (x,value) => x.Base = value),
+                new Tuple<string, Action<SearchIssuesRequest, string>>("user:", (x,value) => x.User = value)
+            };
+
+            foreach (var property in stringProperties)
+            {
+                var request = new SearchIssuesRequest("query");
+
+                // Ensure the specified parameter does not exist when not set
+                Assert.False(request.MergedQualifiers().Any(x => x.Contains(property.Item1)));
+
+                // Set the parameter
+                property.Item2(request, "blah");
+
+                // Ensure the specified parameter now exists
+                Assert.True(request.MergedQualifiers().Count(x => x.Contains(property.Item1)) == 1);
+            }
+        }
+
+        [Fact]
+        public void HandlesDateRangeAttributesCorrectly()
+        {
+            var dateProperties = new List<Tuple<string, Action<SearchIssuesRequest, DateRange>>>()
```

Comment:
Poor old Tuples everyone hates them!  Yep good idea not sure how I ended up with `<List<Tuple<` instead of just a `Dictionary` hahah :grinning: 

These are now tidied up